hm_1/tasks/task_04_02.py

```diff
@@ -0,0 +1,33 @@
+"""
+Write a function that accepts an URL as input
+and count how many letters `i` are present in the HTML by this URL.
+Write a test that check that your function works.
+Test should use Mock instead of real network interactions.
```

Comment:
To ensure that everything works, disable the internet on your machine and run tests.  If it passed that this requirement is fulfilled.